components/scrapingant/scrapingant.app.mjs

```diff
@@ -1,11 +1,91 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "scrapingant",
-  propDefinitions: {},
+  propDefinitions: {
+    url: {
+      type: "string",
+      label: "URL",
+      description: "URL to scrape. This is a required parameter",
+    },
+    browser: {
+      type: "boolean",
+      label: "Browser",
+      description: "Enables using a headless browser for scraping",
+      optional: true,
+    },
+    returnPageSource: {
+      type: "boolean",
+      label: "Return Page Source",
+      description: "Enables returning data returned by the server and unaltered by the browser. Default: false. When true - JS won't be rendered. This feature works only with `browser=true`",
+      optional: true,
+    },
+    cookies: {
+      type: "string",
+      label: "Cookies",
+      description: "Cookies to pass with a scraping request to the target site, i.e.: `cookie_name1=cookie_value1;cookie_name2=cookie_value2`",
+      optional: true,
+    },
+    jsSnippet: {
+      type: "string",
+      label: "JS Snippet",
+      description: "Base64 encoded JS snippet to run once page being loaded in the ScrapingAnt browser. This feature works only with `browser=true`",
+      optional: true,
+    },
+    proxyType: {
+      type: "string",
+      label: "Proxy Type",
+      description: "Specifies proxy type to make request from",
+      options: constants.PROXY_TYPES,
+      optional: true,
+    },
+    proxyCountry: {
+      type: "string",
+      label: "Proxy Country",
+      description: "Specifies the proxy country to make the request from",
+      options: constants.PROXY_COUNTRIES,
+      optional: true,
+    },
+    waitForSelector: {
+      type: "string",
+      label: "Wait for Selector",
+      description: "The CSS selector of the element our service will wait for before returning result. This feature works only with `browser=true`",
+      optional: true,
+    },
+    blockResource: {
+      type: "string[]",
+      label: "Block Resource",
+      description: "Prevents cloud browser from loading specified resource types. Available resource types: `document`, `stylesheet`, `image`, `media`, `font`, `script`, `texttrack`, `xhr`, `fetch`, `eventsource`, `websocket`, `manifest`, `other`. This feature works only with `browser=true`",
```

Comment:
Sounds like these should all be `options` instead of being listed in the description
```suggestion
      description: "Prevents cloud browser from loading the specified resource types",
```